apps/mobile/src/components/DevMenuItems.tsx

```diff
@@ -19,6 +19,12 @@ export function DevMenuItems() {
           navigation.navigate('DesignSystemButtons');
         },
       },
+      {
+        name: 'Debugger',
+        callback: () => {
+          navigation.navigate('Debugger');
+        },
+      },
```

Comment:
can we only display this option when pointing to dev? can't log in as another user in prod